### PR TITLE
Use base64 encoded file instead of multipart upload

### LINKED ISSUE #5: Setup Github actions

### PR DESCRIPTION
**Pull request by hoanhan101**
Also fixes #5

### CODE REVIEW

File: veryfi/client.go [hunk 1]
```diff
@@ -71,7 +81,7 @@ func (c *Client) ProcessDocumentUpload(opts scheme.DocumentUploadOptions) (*sche
 // ProcessDocumentUploadBase64 returns the processed base64 encoded document.
 func (c *Client) ProcessDocumentUploadBase64(opts scheme.DocumentUploadBase64Options) (*scheme.Document, error) {
```
Comment (manycoding) [inline veryfi/client.go]: I think it doesn't make sense to keep this one. The feature is the same as ProcessDocumentUpload

**Review by manycoding — CHANGES_REQUESTED** [review]
I don't see any tests in GitHub actions - do you have any? How do you test it?

**Comment by hoanhan101** [comment]
Tests can be performed locally. Github actions is not setup yet. I'll do that in other PR --> https://github.com/veryfi/veryfi-go/issues/5.

**Comment by manycoding** [comment]
What about my comment?

**Comment by hoanhan101** [comment]
@manycoding Yes I think `ProcessDocumentUploadBase64` should be deleted as well. I'll make PRs for both of these changes today.

**Review by manycoding — APPROVED** [review]

**Pull request merged**